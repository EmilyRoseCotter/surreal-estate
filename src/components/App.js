import React from "react";
import "../styles/App.css";

function App() {
  return (
    <div className="App">
      <body>
        <h2>Surreal estate</h2>
      </body>
    </div>
  );
}

export default App;
