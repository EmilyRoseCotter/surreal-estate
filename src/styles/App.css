@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200&display=swap');

body {
  font-family: 'Nunito Sans', sans-serif;
  margin: 0;
  background-color: lightgrey;
}