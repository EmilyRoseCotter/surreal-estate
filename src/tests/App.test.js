import React from "react";
import { render } from "@testing-library/react";
import App from "../components/App";

it("renders the app", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
