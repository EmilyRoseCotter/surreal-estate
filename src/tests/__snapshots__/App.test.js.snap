// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the app 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h2>
      Surreal estate
    </h2>
  </div>
</DocumentFragment>
`;
